types/components/helpers.d.ts

```diff
@@ -1,7 +1,5 @@
 import * as React from 'react';
 
-export type Omit<T, U> = Pick<T, Exclude<keyof T, keyof U>>;
```

Comment:
Why is this being removed? I think this change is what's causing the CI builds to fail

Reply:
TS 3.5 includes `Omit` as a built-in now